apps/cli/src/commands/send/erc20.ts

```diff
@@ -85,6 +118,8 @@ export default class SendERC20 extends SendBaseCommand<typeof SendERC20> {
 
         const amount =
             this.flags.amount || (await input({ message: "Amount" }));
+        
```

Comment:
Parse the value in this point to bigint with `parseEther`, and remove the extra parsing inside setAllowance.

Reply:
done